spark/src/main/scala/geotrellis/spark/buffer/BufferSizes.scala

```diff
@@ -0,0 +1,6 @@
+package geotrellis.spark.buffer
+
+case class BufferSizes(left: Int, right: Int, bottom: Int, top: Int) {
+  def totalWidth = left + right
```

Comment:
These aren't used and kind of sound like they might be totalWidth and totalHeight of the whole tile, which they are not of course.